### PR TITLE
Header

### PR DESCRIPTION
**Pull request by Aselshine**
Issue #:
[Trello](https://trello.com/invite/b/AklPlVwg/ATTI9fdb094d2c14954c792b9e3930d9949a11CEF7C2/solid-practicum-dominos-team-2)

## PR type

Please check the type of change your PR introduces:

- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Other (please describe):

#### after:
laptop
Ipad
phone adaptive

## Does this introduce a breaking change?

- [ ] Yes (please describe):
- [x] No

## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Accessibility support has been tested
- [x] Responsive layout and functionality have been tested
  - [x] desktop and mobile view responsiveness are supported
  - [ ] no loss of functionality or content when zoomed in to 400%
  - [ ] no horizontal scrolling when zoomed in to 400% (only vertical scrolling, exception: DataGrids)
- [ ] avoids using many levels of prop drilling
- [ ] check console on errors and warnings.
---

References:
- [Clean code](https://github.com/airbnb/javascript)

### CODE REVIEW

File: src/components/Header/Header.jsx [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+  return (
+    <div className={styles.wrapper}>
+
+      <div className={styles.top}>
+        
+        <div className={styles.lng_auth}>
+
+          <Link to="/Contacts" className={styles.contact}>
+            <img width={25} src={phone} alt="" />
```
Comment (aiskairr) [inline src/components/Header/Header.jsx]: Дайте название картинки в атрибуте alt

File: src/components/HomePage/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import React, {useState} from 'react'
+// import Cart from '../Cart/Cart'
```
Comment (aiskairr) [inline src/components/HomePage/HomePage.jsx]: Удалите закомментированный код

File: src/components/HomePage/HomePage.jsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+// import pizza from '../../pizza.json'
```
Comment (aiskairr) [inline src/components/HomePage/HomePage.jsx]: Тут тоже

Comment (aiskairr) [inline src/components/HomePage/HomePage.jsx]: Тут тоже

File: src/components/HomePage/HomePage.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+import Header from '../Header/Header'
+export default function HomePage() {
+  const [clicked, isClicked] = useState(false)
+  return (
+    <div>
+      <Header clicked={clicked} isClicked={isClicked} />
+      {/* <div className="cart-f">
```
Comment (aiskairr) [inline src/components/HomePage/HomePage.jsx]: Тут тоже

File: src/components/HomePage/HomePage.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+        {pizza.map((item) => <Cart key={item.id} {...item} />)}
```
Comment (aiskairr) [inline src/components/HomePage/HomePage.jsx]: Тут тоже

File: src/components/Pages/Auth.jsx [hunk 6]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import styles from './Auth.module.scss'
+import { Link } from 'react-router-dom'
+function Auth() {
+    return (
+        <div>
+  <Link to="/Auth">
+        <h3 className={styles.auth}>Auth</h3>
```
Comment (aiskairr) [inline src/components/Pages/Auth.jsx]: Где i18next?

File: src/components/Header/Header.jsx [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+            <h3 className={styles.phone}>0442 222 11 11</h3>
+          </Link>
+
+          <div className={styles.city}>
+            <img src={placeholder} alt="" />
+            <h3 className={styles.kiev}>Kiev</h3>
```
Comment (aiskairr) [inline src/components/Header/Header.jsx:32]: I18next?

File: src/components/Header/Header.jsx [hunk 8]
```diff
@@ -0,0 +1,112 @@
...
+            </div>
+        </div>
+        <div className={styles.lng_auth}>
+          <select className={styles.ln}>
+            <option value="English">English</option>
```
Comment (aiskairr) [inline src/components/Header/Header.jsx:37]: Используйте i18next

File: src/components/Header/Header.jsx (hunk 1 above)
Comment (Aselshine) [reply src/components/Header/Header.jsx]: Done

File: src/components/Pages/Auth.jsx (hunk 6 above)
Comment (Aselshine) [reply src/components/Pages/Auth.jsx]: Deleted

File: src/components/Header/Header.jsx (hunk 7 above)
Comment (Aselshine) [reply src/components/Header/Header.jsx:32]: I dont use it now, so it was deleted

File: src/components/Header/Header.jsx (hunk 8 above)
Comment (Aselshine) [reply src/components/Header/Header.jsx:37]: Deleted. We dont use it now

File: src/components/HomePage/HomePage.jsx (hunk 4 above)
Comment (Aselshine) [reply src/components/HomePage/HomePage.jsx]: Done

File: src/components/HomePage/HomePage.jsx (hunk 3 above)
Comment (Aselshine) [reply src/components/HomePage/HomePage.jsx]: Done

File: src/components/HomePage/HomePage.jsx (hunk 2 above)
Comment (Aselshine) [reply src/components/HomePage/HomePage.jsx]: Done

**Comment by Esenushka** [comment]
Измените скриншоты так как у вас изменилисась адаптика

**Comment by Aselshine** [comment]
> 

apaptive for Web
phone adaptive
adaptive for Ipad

**Comment by Aselshine** [comment]
adaptive for iphone
apaptive for Web
adaptive for Ipad

**Review by Esenushka — APPROVED** [review]

**Pull request merged**